src/data-loader.ts

```
@@ -814,3 +846,128 @@ export async function loadMonthlyUsageData(
+		cost: number;
+		sessionKey: string;
+		model: string | undefined;
+	}>,
+): Map<string, WindowUsage> {
+	const windowMap = new Map<string, WindowUsage>();
+
+	for (const entry of allEntries) {
+		const windowId = get5HourWindowId(entry.timestamp);
+		const month = entry.timestamp.substring(0, 7);
+
+		let window = windowMap.get(windowId);
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)